### DISCUSSION TITLE
error after update

### DISCUSSION THREAD

**Discussion by lidas1234** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1759] [General]
Hi I did an update of my display TFT35 V3, but now every now and then I find myself this error and solve it by turning off the printer and turning it back on, do you have any ideas to advise me to do
Thanks
![IMG_2285](https://user-images.githubusercontent.com/53397523/111917585-05580180-8a81-11eb-8a93-faf46c765de0.jpeg)

**Comment by oldman4U** [comment]
What happens when you press Confirm?

Do you have a filament sensor attached?

Have you tried to remove the TFT from the enclosure it is mounted to?

**Comment by lidas1234** [comment]
Nothing happens when I press Confirm

I have a filament sensor connected

I have not tried to remove the TFT from the case it is mounted on

**Reply by oldman4U** [reply]
What happens when you press Pause during a print?

I would check that you have all functions enabled in Marlin which are mentioned in config.ini and use a Marlin nightly build. Most likely you can resume the print after that.

Remove the screen to see if the problem itself is solved then.

**Comment by oldman4U** [comment]
Remove the display from the enclosure.

Make sure you have Marlin configured according to what's written in config.ini.

If this doesn't help remove the sensor physically and from firmware.

**Comment by oldman4U** [comment]
Any news?
